gitbook-docs/streaming_api.md

```diff
@@ -50,3 +50,31 @@ The server MAY provide:
 `name`, `self` and `roles` MUST return the same values as provided in the `swname`, `self` and `roles` properties within the [DNS-SD advertisement](connection.md) (if implemented).
 
 `version` MUST be the same value as `version` within the associated endpoints list provided by the http `GET` request to `/signalk` within the [REST API](rest_api.md) (if implemented).
+
+## History playback
+
+The server MAY support history playback from a certain point in time with a specified rate.
+
+To create a WebSocket connection that plays back data the client uses the request parameter `startTime` to specify the start timestamp and the optional request parameter `playbackRate` to specify the rate. Rate value parameter is a floating point value with value `1` equal to real time playback and for example `0.5` to half the real time rate and `5` to five times real time rate. Omitting the `playbackRate` will result in real time playback.
+
+The playback api is located at `/signalk/v1/playback`. An example url for history playback streaming: `wss://localhost:3443/signalk/v1/playback?subscribe=self&startTime=2018-08-24T15:19:09Z&playbackRate=5`.
```

Comment:
It should also have an endTime, so you can select a specific time period, rather than playing till you reach now(). Also `/signalk/v1/playback/` is equivalent to `/signalk/v1/stream/` not `/signalk/v1/api` isnt it?, eg /stream does a ws endpoint, /api does an http rest reply.

Reply:
The client can drop the connection whenever it wants, so endTime is not strictly needed.

Reply:
Sorry, missed your point regarding the paths?